test/tools/isoltest.cpp

```diff
@@ -213,8 +214,23 @@ TestTool::Result TestTool::process()
 	}
 }
 
+void TestTool::updateTestCase()
+{
+	ofstream file(m_path.string(), ios::trunc);
+	m_test->printSource(file);
+	m_test->printUpdatedSettings(file);
+	file << "// ----" << endl;
+	m_test->printUpdatedExpectations(file, "// ");
+}
+
 TestTool::Request TestTool::handleResponse(bool _exception)
 {
+	if (!_exception && m_options.yesToAll)
+	{
+		updateTestCase();
+		return Request::Rerun;
```

Comment:
This will spin, if updating fails, right :-)? For example if you add a test that behaves differently between ir codegen and legacy codegen? But well, maybe still fine and not worth complicating this to guard against that...

Reply:
(also in such cases ``yes u | isoltest`` would also spin anyways)